code/__DEFINES/_click.dm

```diff
@@ -0,0 +1,34 @@
+//Defines file for byond click related parameters
+//this is mostly for ease of use and for finding all the things that use say RIGHT_CLICK rather then just searching "right"
+
+
+//Mouse buttons pressed/held/released 
+#define RIGHT_CLICK "right"
```

Comment:
no wonder shit broke, you guys are merging stuff that was only possible due to combat mode @Kitsunemitsu